components/Layout/Navigation.js

```diff
@@ -26,6 +26,8 @@ class Navigation extends React.Component {
       <nav className="mdl-navigation" ref="root">
         <Link className="mdl-navigation__link" to="/">Home</Link>
         <Link className="mdl-navigation__link" to="/about">About</Link>
+        <Link className="mdl-navigation__link" to="/blog/2016/07/01/Post-1">Post 1</Link>
+        <Link className="mdl-navigation__link" to="/blog/2016/07/01/Post-2">Post 2</Link>
```

Comment:
Also a fan of following Jekyll's URL conventions, it will help with portability and converting older blogs to the hot new tech in town. But per the convention, it should be an explicit convention that could then be extrapolated to a path like 2016-07-12-parse-all-the-things turns into 2016/july/parse-all-the-things with an option to override from front matter if an explicit url front-matter is given.